portal/server/helpers/filterReportJsonToRelevantKeys.js

```diff
@@ -0,0 +1,23 @@
+import { UTILISATION_REPORT_HEADERS } from '@ukef/dtfs2-common';
+
+/**
+ * Iterates through an array of objects and removes any key value pairs
+ * from each object where the key isn't in the UTILISATION_REPORT_HEADERS const
+ * @param {Array<Object>} reportJson auth token
```

Comment:
auth token?

Reply:
how on earth did I get that in there, nice catch aha